app/uploaders/file_uploader.rb

```diff
@@ -7,12 +7,6 @@ class FileUploader < CarrierWave::Uploader::Base
   # include CarrierWave::RMagick
   # include CarrierWave::MiniMagick
 
-  # Override the directory where uploaded files will be stored.
-  # This is a sensible default for uploaders that are meant to be mounted:
-  def store_dir
-    "#{store_prefix}uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
-  end
-
   def store_prefix
```

Comment:
Thanks @rdeskins but I think what we really what to do is to get rid of store prefix